test/extensions/filters/http/grpc_http1_bridge/http1_bridge_filter_test.cc

```diff
@@ -69,13 +71,44 @@ TEST_F(GrpcHttp1BridgeFilterTest, NoCluster) {
   Http::TestResponseHeaderMapImpl response_headers{{":status", "404"}};
 }
 
+TEST_F(GrpcHttp1BridgeFilterTest, Http2HeaderOnlyResponse) {
+  protocol_ = Http::Protocol::Http2;
+
+  Http::TestRequestHeaderMapImpl request_headers{
+      {"content-type", "application/grpc"},
+      {":path", "/lyft.users.BadCompanions/GetBadCompanions"}};
+
+  ON_CALL(*decoder_callbacks_.cluster_info_, statsScope())
```

Comment:
Should this go in the constructor of GrpcHttp1BridgeFilterTest? I'm not sure if it will cause issues in the tests that you didn't add it to.